corehq/messaging/scheduling/tasks.py

```diff
@@ -218,10 +218,7 @@ def create_new_instance_for_recipient(self, recipient_type, recipient_id):
                 recipient_type,
                 recipient_id
             )
-
-            if self.action_definition.reset_case_property_name:
-                handle_case_alert_schedule_instance_reset(instance, self.schedule, self.reset_case_property_value)
-
+            self.handle_existing_instance(instance)
```

Comment:
This bit of duplication removal is unrelated to the docs in the following commit. I found it while investigating and forming the theory of operation for conditional alert rule processing.